api/express-gen-ts/env/test.env

```diff
@@ -0,0 +1,6 @@
+# Environment
```

Comment:
I don't think we need a testing environment. dev and prod make sense. If we want to spin up a test server we'd just spin up the prod instance in k8's in a different namespace.